_episodes/05-exporting-data-2.md

```diff
@@ -0,0 +1,132 @@
+compiling Excel-stored data from multiple sources. There’s dates in each file- Excel interprets them as their own internally consistent
+serial numbers. When you combine the data, Excel will take the serial number from the place you’re importing it from, and interpret it
+using the rule set for the version of Excel you’re using. Essentially, you could be adding errors to your data, and it wouldn’t
+necessarily be flagged by any data cleaning methods if your ranges overlap.
+
+Storing data in a universal, open, and static format will help deal with this problem. Try tab-delimited (tab separated values
+or TSV) or comma-delimited (comma separated values or CSV). CSV files are plain text files where the columns are separated by commas,
+hence 'comma separated values' or CSV. The advantage of a CSV file over an Excel/SPSS/etc. file is that we can open and read a CSV file
+using just about any software, including plain text editors like TextEdit or NotePad. 
+Data in a CSV file can also be easily imported into other formats and
+environments, such as SQLite and R. We're not tied to a certain version of a certain expensive program when we work with CSV files, so
+it's a
+good format to work with for maximum portability and endurance. Most spreadsheet programs can save to delimited text formats like CSV
+easily, although they may give you a warning during the file export.
+
+To save a file you have opened in Excel in CSV format:
+
+1. From the top menu select 'File' and 'Save as'.
+2. In the 'Format' field, from the list, select 'Comma Separated Values' (`*.csv`).
+3. Double check the file name and the location where you want to save it and hit 'Save'.
+
+An important note for backwards compatibility: you can open CSV files in Excel!
+
+![Saving an Excel file to CSV](../fig/excel-to-csv.png)
+
+## A Note on Cross-platform Operability
+
+By default, most coding and statistical environments expect UNIX-style line endings (`\n`) as representing line breaks.  However, Windows uses an alternate line ending signifier (`\r\n`) by default for legacy compatibility with Teletype-based systems.  
+
+As such, when exporting to CSV using Excel, your data in text format will look like this:
+
+>data1,data2\r\n1,2\r\n4,5\r\n
+
+When opening your CSV file in Excel again, it will parse it as follows:
+
+<img width="307" alt="screen shot 2017-03-31 at 7 15 07 pm" src="https://cloud.githubusercontent.com/assets/13110354/24560663/db4a5786-1643-11e7-931a-ca2c72336878.png">
+
+However, if you open your CSV file on a different system that does not parse the "\r" it will interpret your CSV file differently:
+
+Your data in text format then look like this:
+
+>data1 <br>
+>data2\r<br>
+>1<br>
+>2\r<br>
+>…
+
+This will then in turn parse as:
+
+<img width="308" alt="screen shot 2017-03-31 at 7 26 42 pm" src="https://cloud.githubusercontent.com/assets/13110354/24561066/a990327c-1645-11e7-90b5-35a44e90f8d9.png">
```

Comment:
using a more descriptive alt text would be useful

Reply:
Definitely. Will do.